src/Metadata/Property/PropertyMetadata.php

```diff
@@ -31,8 +31,9 @@
     private $identifier;
     private $childInherited;
     private $attributes;
+    private $nestedOperation;
```

Comment:
I don't think nestedOperation is a good name, but then again readableLink / writableLink are very confusing names as well...

Perhaps `$subcollection` / `hasSubcollection`?